api/src/transactions.rs

```diff
@@ -38,7 +38,7 @@ use aptos_types::{
     vm_status::StatusCode,
     AptosCoinType, CoinType,
 };
-use aptos_vm::{AptosSimulationVM, AptosVM};
+use aptos_vm::{AptosSimulationVM, AptosVm};
```

Comment:
I think VM makes sense, since it's not one word but abbreviation

Reply:
I think main motivation is when you have `AptosVmBlockExecutor` this reads nicer than `AptosVMBlockExecutor` so hence the change, I think in one of @igor-aptos's PR we discussed that some time ago. But I do not have a string opinion, would you prefer `VM` over `Vm`?

Reply:
There is also things like VMError etc, we should most likely do a separate PR even if we do change to Vm